care/apps/patients/serializers.py

```diff
@@ -341,7 +352,7 @@ def get_patient_contact_number(self, instance):
         return instance.patient.phone_number
 
     def get_patient_relation(self, instance):
-        return instance.patient_family.relation if instance.patient_family else "Self"
+        return instance.relation if instance.relation else "Self"
```

Comment:
"Self" should be populated in relation field too. So there should not needed any conditional statement